docs/source/user-guide/example-usage.md

```diff
@@ -40,7 +42,7 @@ async fn main() -> datafusion::error::Result<()> {
   ctx.register_csv("example", "tests/example.csv", CsvReadOptions::new()).await?;
 
   // create a plan to run a SQL query
-  let df = ctx.sql("SELECT a, MIN(b) FROM example GROUP BY a LIMIT 100").await?;
+  let df = ctx.sql("SELECT \"A\", MIN(b) FROM example GROUP BY \"A\" LIMIT 100").await?;
```

Comment:
I am not sure about changing this code example. When I tried this example code now using https://github.com/apache/arrow-datafusion/blob/master/datafusion/core/tests/example.csv

I get an error
```
     Running `/Users/alamb/Software/target-iox2/debug/df_test`
Error: SchemaError(FieldNotFound { qualifier: None, name: "A", valid_fields: Some(["example.a", "example.b", "example.c"]) })
```
Maybe we should add an explicit link to the example.csv file ?